workers/unity/Packages/com.improbable.gdk.core/MutableView/MutableView.cs

```diff
@@ -245,7 +245,11 @@ public bool TryGetEntity(long entityId, out Entity entity)
                         return;
                     }
 
-                    AddComponent(entity, new AuthorityLossImminent<T>());
+                    AddComponent(entity, new AuthorityLossImminent<T>
+                    {
+                        AuthorityLossAcknowledged = false,
```

Comment:
Why have both of these? I would have thought only the first is needed?

Reply:
AuthorityLossImminent isn't one of the components which gets cleaned up after a tick so if we only have AuthorityLossAcknowledged variable then we don't know whether it has already been acknowledged or not.

Reply:
Can have another component added as a tag saying it's been acked. It's an easier to use API if there is only one field on the component. Especially if they have similar names, someone will set the wrong one and it will take ages to notice.